Makefile

```diff
@@ -326,9 +326,9 @@ test_p2p:
 test_p2p_addrbook:
 	go test -run AddrBook -v -count=1 ./p2p/...
 
-.PHONY: lint
+.PHONY: go_lint
```

Comment:
Can you move this next to the other `go_*` targets?

Right now its just in between all the unit tests.

![Screen Shot 2022-10-03 at 3 10 44 PM](https://user-images.githubusercontent.com/1892194/193695085-5a8cdc34-de0d-47e7-936a-922b186fd0ba.png)